ios/Runner/Info.plist

```diff
@@ -41,5 +43,8 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<!-- TODO verify this API usage and explanation -->
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>The app required access to the photo library for the selection of a profile picture which will be used and shown on your public CollAction profile</string>
```

Comment:
It should be present tense.

Photo library access is required for selecting one photo which will be used on the users public profile

Or something similar.